src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebVersionRouteConfig.kt

```diff
@@ -28,6 +28,10 @@ object WebVersionRouteConfig : RouteConfig
                     method = HttpMethod.post)
                     .json()
                     .secure(reviewReports),
+            WebEndpoint("/reports/:name/versions/:version/data/:data/",
```

Comment:
That csv data endpoint was just missing before? Why do we have a csv content and a binary content endpoint in the web routing (as opposed to the api routing)?

Reply:
Yeah maybe we don't need this. The integration test was sending an "accepts" csv header, but I guess a browser will always send a more permissive one, so I suppose could just make the test also more permissive.

Reply:
Although this does make me think that the api route won't work for clients who specify they only accept csvs. I will add a test in the api for this.